utm_localization_topic_name: /localization/utmpose

min_dis: 0.05

node_rate: 50   # [Herz]
